packages/core/stdlib/datetime/native.ts

```diff
@@ -1,14 +1,11 @@
+import IScope from "../../lib/interfaces/IScope";
+
 const isNullish = (v) => v === undefined || v === null;
+// TODO: isNullish is also declared on the utils file, what are the difference between them and why the are named the same
```

Comment:
oh, both of them are the same. I just forgot to use it from the utils file. We can actually just import that instead of rewriting it here